### PR TITLE
Features

### PR DESCRIPTION
**Pull request by Tresor11**
this a clone of the new york times web site

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,365 @@
...
+  <body>
+     <nav>
+       <div  class="navbar">
+         <div class="icon">
+            <i class= "fa fa-bars"></i>
+            <i class="fa fa-search"></i>
+            <a href="#"> SPACE & COSMOS </a>
+         </div>
+         <div class="title">
+            <img src="./images/title.PNG" alt="logo" id="newyork">
```
Comment (shshamim63) [inline index.html]: The logo should be enclosed inside a link.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,365 @@
...
+                  
+                  “It was a very special moment, and one we took very seriously as scientists,” said Dr. Kovac, who chose his words as carefully as he tended his radio telescopes. <br> <br>
+                  
+                  Andrei Linde of Stanford, a prolific theorist who first described the most popular variant of inflation, known as chaotic inflation, in 1983, was about to go on vacation in the Caribbean last week when Chao-Lin Kuo, a Stanford colleague and a member of Dr. Kovac’s team, knocked on his door with a bottle of Champagne to tell him the news. <br> <br>
+                              
+               </p>
+               <iframe width="600" height="319" src="https://www.youtube.com/embed/ZlfIVEy_YOA"  allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Comment (shshamim63) [inline index.html]: Please remove inline CSS.

**Review by shshamim63 — COMMENTED** [review]
## :point_right:  Changes Required! :point_left: 
___

Hi @Tresor11  and @Datagirlcmr ,
Well done :clap:, you have covered of the part of this project. But there are few things that you have to modify to get approval for this project.
 ##### Please Follow the instructions below :point_down: to declare this project as complete.
___

#### [Mandatory Changes] :clubs: 
1. Well the navbar logo should be a link. 
2. Your navbar item are not aligned properly. Log in button's text is not in inline. Left part of the navbar needs a little bit of margin on the top side. `Subscribe for full access` is almost attached to the bottom of the navbar section. Please align that too.
image1
3. Item of main body is collapsing. Image and text are rendering one above another. Kindly fix this issue.
image2
4. There is a merge conflict on your PR. Kindly fix it.
___
### [Optional Changes]
- Well i think the font size of main part is large, maybe font size 20px and line height 30px can be more appealing.
---
### Please fix the mentioned issues and submit another code review.

## Happy Coding :four_leaf_clover:
### Shakhawat Hossain
[Linkedin](https://www.linkedin.com/in/shakhawat-hossain-7750b0127/) | [Slack](Shakhawat Hossain)

**Review by Redvanisation — APPROVED** [review]
# Project approved ✔️ 

### Hello team 🌟 

### _Nice work on this project, it seems that you guys have met the requirements so I am approving your project. 👍_ 

### You may merge to master and submit a project completion form. 🚀 

### Good luck! 🍀

**Pull request merged**